docs/rpc-compatibility.md

```diff
@@ -0,0 +1,295 @@
+# RPC Updates to the bcoin
+
+## Ports
+Bitcoin Core does not separate wallet and node:
+
+| Network | Bcoin Node RPC | Bcoin Wallet RPC | Bitcoin Core (Wallet & Node)|
+| ---     | ---            | ---              | ---         |
+| Main    | 8332           | 8334             | 8332        |
+| Testnet | 18332          | 18334            | 18332       |
+| Regtest | 48332          | 48334            | 18443       |
+
+# List of RPC Calls in Bitcoin Core vs Bcoin
+This is a list of commands existing in core and bcoin.  
+This list currently does not include compatibility information.
+
+Note: zmq is separate from bcoin, see [bzmq](https://github.com/bcoin-org/bzmq/).
+
+## Node RPC calls
+
+### Server - control
+These are same for wallet and the node.
+
+| RPC Method   | Bcoin              | Core                                                  | Compatible |
+| ----         | ----               | ----                                                  | ----       |
+| `getrpcinfo` | `-`                | :heavy_check_mark: [since v0.18] [PR][new-getrpcinfo] | :x:        |
+| `help`       | :heavy_check_mark: | :heavy_check_mark:                                    | :x:        |
+| `stop`       | :heavy_check_mark: | :heavy_check_mark:                                    | :question: |
```

Comment:
Why aren't `help` and `stop` compatible with Core?

Reply:
basically the way help returns differently formatted helps, that's it. Let's say if someone was using parser to parse Core help message, it would not work with bcoin.

`stop` is probably compatible will tests.